tests/python/unittest/test_executor.py

```diff
@@ -140,22 +140,20 @@ def test_dot():
 
 def test_reshape():
     x = mx.sym.Variable('x')
-    y = mx.sym.FullyConnected(x, num_hidden=4)
+    y = mx.sym.Dropout(x, p=0.2)
```

Comment:
+1

Reply:
it was modified because FullyConnected originally output an MKLDNN array. It didn't work with reshape. With the new design of NDArray, it's not a problem any more. I'll revert the changes here.